pkg/kubelet/kubelet_pods.go

```diff
@@ -472,6 +472,15 @@ func (kl *Kubelet) GenerateRunContainerOptions(ctx context.Context, pod *v1.Pod,
 	if err != nil {
 		return nil, nil, err
 	}
+
+	if kl.draManager != nil {
+		resOpts, err := kl.draManager.GetContainerInfo(pod, container)
```

Comment:
A better name for this function as this level is probably `GetResources()` (just like we have for the `containerManager`).